lib/ui/views/home/widgets/restaurant_card/restaurant_card_model.dart

```diff
@@ -1,10 +1,19 @@
 import 'package:stacked/stacked.dart';
+import 'package:stacked_services/stacked_services.dart';
+import 'package:veggie_go_malaysia/app/locator.dart';
+import 'package:veggie_go_malaysia/app/router.dart';
 
 class RestaurantCardModel extends FutureViewModel {
+  final NavigationService _navigationService = locator<NavigationService>();
+
   Future<String> calculateDistanceFromUser() async {
     return '1.2km';
```

Comment:
i think this will be a getter of the restaurant object, and not inside the viewmodel

Reply:
the api should return this information if we are already sending querying using the user location?

Reply:
@fremfi yeah we discussed this on slack, it will be changed so that it is encapsulated with the restaurant class, not in viewmodels